src/scripting/KM_ScriptingStates.pas

```diff
@@ -507,6 +511,31 @@ function TKMScriptStates.GameTime: Cardinal;
 end;
 
 
+//* Version: 7000+
+//* Returns an array with IDs of teams for all players
+//* Result: Array of team IDs
```

Comment:
It's not clear what this function returns. Is it a mapping (lookup table) from Hand Index to Team? (that would make the most sense for the script). In that case you should iterate over hands count not NetPlayers count (so the length of the returned array is always hands count).

Otherwise it's unclear how this function could be useful to the script, since the indexes of the array will not match up with player indexes that the script uses in other functions (e.g. the player/hand index in Actions.OverlayTextSet would not be the same as the index into the array returned by this function).